src/scss/components/main-nav/_main-nav.scss

```diff
@@ -1,5 +1,7 @@
 @charset "UTF-8";
 
+$saa-color-vivid-red: #e50808;
```

Comment:
@katrialesser It appears due to the order in which the files are compiled, the main navigation partial within components is not able to reference the color variables within utilities. 

I've temporarily added the color to the top of the main-nav partial for now. I'd love to hear your thoughts on how we should restructure this. 😄

Reply:
Fixed with utilities index reference/